.platform.app.yaml

```diff
@@ -74,6 +74,7 @@ hooks:
         rm blackfire.tar.gz blackfire blackfire.sha1
     )
 
+    pip install blackfire-1.5.5_internal.14-cp39-cp39-manylinux2010_x86_64.whl
```

Comment:
I guess we'll remove this line once the new version is out?

Reply:
Yes